birdhouse/optional-components/README.rst

```diff
@@ -0,0 +1,140 @@
+Optional components
+===================
+
+Monitor all components in CANARIE node, both public and internal url
+--------------------------------------------------------------------
+
+So that the url ``https://<PAVICS_FQDN>/canarie/node/service/stats`` also return
+what the end user really see (a component might work but is not accessible to
+the end user).
+
+This assume all the WPS services are public.  If not the case, make a copy of
+this config and adjust accordingly.
+
+How to enable this config in ``env.local`` (a copy from
+`env.local.example <birdhouse/env.local.example>`_):
```

Comment:
Same problem with broken relative link when browsing on github.

Reply:
I need to find a better way for linking to non-rst files. I feel like I'm close.